### PR TITLE
fix(challenges): Remove duplicate Rosetta Code sorting problems

### LINKED ISSUE #34776: Multiple versions of sorting algorithm challenges
A little while ago I reviewed some new Rosetta Code challenges for the Coding Interview Prep section. Some of the new challenges were sorting challenges (bubble, insertion, and so on) that already exist in another part of the Coding Interview Prep section, and @ezioda004 brought up a good point about the duplicate challenges being excessive.

Here's the PR for reference where we discuss some possible solutions: https://github.com/freeCodeCamp/freeCodeCamp/pull/34634

What do you all think? Would be great to get some feedback about what everyone thinks is best to do in this situation.

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

This PR is to remove duplicate Rosetta Code sorting problems that exist in other parts of the curriculum. Here are the problems that are being removed: 
* Sorting algorithms/Bubble sort
* Sorting algorithms/Counting sort
* Sorting algorithms/Heapsort
* Sorting algorithms/Insertion sort
* Sorting algorithms/Merge sort
* Sorting algorithms/Quicksort
* Sorting algorithms/Radix sort
* Sorting algorithms/Selection sort

This PR only removes the challenges in English. I could remove the other languages in this PR, or in another PR.

Closes #34776

### CODE REVIEW

**Comment by RandellDawson** [comment]
@scissorsneedfoodtoo Just to confirm, I see 7 challenges listed above, but see 8 files being removed.

**Comment by scissorsneedfoodtoo** [comment]
@RandellDawson, sorry about that! Thanks for pointing that out. Just added the missing problem.

**Review by thecodingaviator — APPROVED** [review]
LGTM! 👍 Looks good to be merged to me. Waiting for a review from @raisedadead

**Review by raisedadead — APPROVED** [review]

**Pull request merged**
